apitools/base/py/credentials_lib.py

```diff
@@ -331,6 +334,9 @@ def _WriteCacheFile(self, cache_filename, scopes):
                         # If it's not locked, the locking process will
                         # write the same data to the file, so just
                         # continue.
+                except:  # pylint: disable=bare-except
```

Comment:
It would require mocking locked_file which was removed from more recent versions of oauth2client; this code is simple enough that I'd prefer not to unless you object.